tests/CourseDashboardTests.cpp

```diff
@@ -1,8 +1,25 @@
 #include <gtest/gtest.h>
 #include "CourseDashboard.hpp"
+#include "User.hpp"
 
+class CourseDashboardTests : public ::testing::Test
+{
+protected:
+    CourseDashboard testCdb;
+    User testU1;
+
+    CourseDashboardTests()
+    {
+        testU1 = User("Szymon", "SzymonGajewski", "wieczorowa", "SzymonGajewski", "SzymonGajewski");
+        testCdb.createUser(testU1);
+        testCdb.retriveUserByNick("SzymonGajewski");
+        testCdb.deleteUserByNick("SzymonGajewski");
+        EXPECT_EQ("SzymonGajewski", testU1.getNick());
```

Comment:
this EXPECT_EQ doesn't test anything in the CourseDashboard. It only checks if the User nick was set properly. This is not a test of CourseDashboard.

Reply:
Yes it's true, These EXPECTs weren't tests.

Reply:
Changed on ASSERT_TRUE.